boolean/test_boolean.py

```diff
@@ -180,9 +249,9 @@ def test_eval(self):
         a = boolean.Symbol("a")
         self.assertTrue(~a == ~a)
         self.assertFalse(a == boolean.parse("~~a", eval=False))
-        self.assertTrue(a == ~~a)
-        self.assertTrue(~a, ~~~a)
-        self.assertTrue(a, ~~~~a)
+        self.assertTrue(a == ~ ~a)
```

Comment:
Actually this change in this test is a typo/mishap I need to fix: its should not have been  changed
But anything goes: with spaces or no spaces: the tokenizer is after all the Python tokenizer so it has the same behaviour